data/json/mapgen/house_garage6.json

```diff
@@ -0,0 +1,148 @@
+[
+	{
+		"comment": "two bedroom one bath single garage home.",
+		"type": "mapgen",
+		"om_terrain": [
+			"house"
+		],
+		"method": "json",
+		"weight": 100,
+		"object": {
+			"rows": [
+				"..........SSS.SSSSSS....",
+				"...dzzzzd..S.PSSSSSSz...",
+				"..--WWWW--zSz-MMMMMM--..",
+				"..|^!!!!^|~x~|      X|..",
+				"..|!!CC!!|bSb|       |..",
+				".zW!!TT!!WzSzW       |z.",
+				".zW!!CC!!WzSzW       *S.",
+				"..|!!!!!!|fSf|      >|R.",
+				"..|-<<!!!|fSf|      >|..",
+				"..|F!!!!!WzSzW      >Wz.",
+				".b|O!!<!!WzSzW      >Wz.",
+				".zWI!!<!!|dSd|L 34>>>|..",
+				".b|E!!!!!|zSz--+------..",
+				"..|-<<<!!|G*G|K ^|I|;|f.",
+				".d|^             +!!!Wz.",
+				".S*         D|-+-|BB!|f.",
+				".d| D??  ----|   ---+|..",
+				".zW ?    | :K| C     |..",
+				".zW ? T  +   |DD   @@Wz.",
+				"..|      |C  ---   @@Wz.",
+				"..|^    ^|D  @@|:   K|..",
+				"..--GAAG---W-W----W---..",
+				"...SSSSSS.bz.zb...z.....",
+				"....SSSS................"
+			],
+			"terrain": {
+				" ": "t_floor",
+				"!": "t_linoleum_white",
+				".": "t_grass",
+				"-": "t_wall",
+				"|": "t_wall",
+				":": "t_floor",
+				";": "t_linoleum_white",
+				"#": "t_floor",
+				"*": "t_door_locked",
+				"+": "t_door_c",
+				">": "t_floor",
+				"<": "t_linoleum_white",
+				"?": "t_floor",
+				"@": "t_floor",
+				"^": "t_floor",
+				"3": "t_floor",
+				"4": "t_floor",
+				"z": "t_shrub",
+				"b": "t_dirt",
+				"d": "t_dirt",
+				"f": "t_dirt",
+				"A": "t_door_glass_c",
+				"B": "t_linoleum_white",
+				"C": "t_floor",
+				"D": "t_floor",
+				"E": "t_linoleum_white",
+				"F": "t_linoleum_white",
+				"G": "t_wall_glass",
+				"H": "t_floor",
+				"I": "t_linoleum_white",
+				"K": "t_floor",
+				"L": "t_floor",
+				"M": "t_door_metal_locked",
+				"O": "t_linoleum_white",
+				"P": "t_grass",
+				"R": "t_grass",
+				"S": "t_sidewalk",
+				"T": "t_floor",
+				"W": "t_window_domestic",
+				"X": "t_gates_mech_control",
+				"~": "t_fence_h",
+				"x": "t_fencegate_c"
+			},
+			"furniture": {
+				"@": "f_bed",
+				"K": "f_bookcase",
+				">": "f_counter",
+				"<": "f_counter",
+				"#": "f_rack",
+				":": "f_dresser",
+				"3": "f_washer",
+				"4": "f_dryer",
+				"^": "f_indoor_plant",
+				"?": "f_sofa",
+				"b": "f_bluebell",
+				"d": "f_dahlia",
+				"f": "f_dandelion",
+				"I": "f_sink",
+				"T": "f_table",
+				"B": "f_bathtub",
+				"C": "f_chair",
+				"D": "f_desk",
+				"R": "f_trashcan",
+				"E": "f_cupboard",
+				"F": "f_fridge",
+				"O": "f_oven",
+				"L": "f_locker",
+				"H": "f_armchair",
+				"P": "f_sign"
+			},
+			"toilets": {
```

Comment:
Remove the section if it's empty.
But should it really be empty? A house with two bedrooms but no toilet?